api/src/main/clojure/xtdb/datalog.clj

```diff
@@ -41,7 +41,9 @@
         (assoc :basis-timeout (Duration/ofSeconds 1))))"
   ^java.util.concurrent.CompletableFuture
   [node q & args]
-  (-> (impl/open-datalog& node (-> q (update :basis impl/after-latest-submitted-tx node)) args)
+  (-> (impl/open-datalog& node (->> (update q :basis impl/after-latest-submitted-tx node)
```

Comment:
subjective: when you've got fns like `into` or `merge`, where the threaded argument could legitimately go as either the first or last argument, I prefer not to thread, so that it's visually obvious whether you're defaulting or overriding - with a thread, I find myself mentally unwinding it to figure out what it's doing :sweat_smile: